org.eclipse.jdt.core.manipulation/core extension/org/eclipse/jdt/internal/corext/util/TypeFilter.java

```diff
@@ -113,7 +113,7 @@ private IEclipsePreferences getDefaultPreferenceStore() {
 
 	private String getPreference(String key, String def) {
```

Comment:
While I don't mind formatting changes in general. This one creates just noise as it doesn't look related to the rest of the PR and no other change is done to the file.